streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -1361,6 +1369,10 @@ public void start(Application appParam, boolean auto) throws Exception {
         } else {
             if (ExecutionMode.isKubernetesApplicationMode(application.getExecutionMode())) {
                 AssertUtils.state(buildResult != null);
+                ParameterTool parameter = ParameterTool.fromPropertiesFile(String.format("%s/conf/flink-conf.yaml", flinkEnv.getFlinkHome()));
```

Comment:
hi MonsterChenzhuo:
In the console layer, we currently avoid using the flink api directly, You can refer to method `expire` of `SavePointServiceImpl.java`:
```
 FlinkEnv flinkEnv = flinkEnvService.getByAppId(application.getAppId());
        AssertUtils.state(flinkEnv != null);
         String fsPath = Integer.parseInt(
            flinkEnv.convertFlinkYamlAsMap()
                .getOrDefault(JobManagerOptions.ARCHIVE_DIR.key(), "...")
        );
```
...